### PR TITLE
runeliteapi: add timeout to rlp api call.

### PR DESCRIPTION
**Pull request by f0rmatme**
Updated okhttp3 from 3.7.0 to okhttp3 4.0.0
Added timeout stuff

Tested on my fast connection, !price seems faster/less/no microstutter
Needs to be tested on a slow connection, justin was talking about how he had a firewall and it would freeze his client for 15 seconds. Maybe he can test.

### CODE REVIEW

**Comment by zeruth** [comment]
Apply this to RL_CLIENT as well. They were complaining about !price etc too.

**Comment by Kyleeld** [comment]
> Apply this to RL_CLIENT as well. They were complaining about !price etc too.

Added

**Comment by f0rmatme** [comment]
I made the timeout higher for runelites api as kc commands and lookup commands maybe take longer to load

File: http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java [hunk 1]
```diff
@@ -103,6 +103,9 @@
 
 		CLIENT = new OkHttpClient.Builder()
 			.pingInterval(30, TimeUnit.SECONDS)
+			.connectTimeout(5655, TimeUnit.MILLISECONDS)
```
Comment (Kyleeld) [inline http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java]: duplicate line?

File: http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java [hunk 2]
```diff
@@ -119,6 +122,9 @@ public Response intercept(Chain chain) throws IOException
 
 		RLP_CLIENT = new OkHttpClient.Builder()
 			.pingInterval(30, TimeUnit.SECONDS)
+			.connectTimeout(5655, TimeUnit.MILLISECONDS)
```
Comment (Kyleeld) [inline http-api/src/main/java/net/runelite/http/api/RuneLiteAPI.java]: duplicate line.

**Comment by SRLJustin** [comment]
I can test the proxy/firewall connection next week when I am back on site

**Comment by zeruth** [comment]
We will need some more investigation into this later, but for now, we'll merge for update and see if we can get some good feedback from those affected.

**Pull request merged**